docs/index.rst

```diff
@@ -50,7 +50,7 @@ Installing DEPOT can be done from PyPi itself by installing the ``filedepot`` di
 
 Keep in mind that DEPOT itself has no dependencies, if you want to use GridFS storage,
 S3 or any other storage that requires third party libraries, your own application is
-required to install the dependency. In this specific case ``pymongo`` and ``boto``
+required to install the dependency. In this specific case ``pymongo`` , ``google-cloud-storage``` and ``boto``
```

Comment:
The paragraph talks explicitly about `GridFS` and `S3`, so that's why only `pymongo` and `boto` were mentioned.

I would omit `google-cloud-storage` dependency from here, or rewrite the paragraph to cover all three of them